core/orch_test.go

```diff
@@ -1248,8 +1252,15 @@ func TestPriceInfo_ReturnsBigRat(t *testing.T) {
 	orch := NewOrchestrator(n, nil)
 
 	priceInfo, err := orch.PriceInfo(ethcommon.Address{})
-	assert.Nil(t, err)
-	assert.Zero(t, expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	assert.Nil(err)
+	assert.Zero(expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	fixedPrice, err := common.PriceToFixed(expPricePerPixel)
+	expPrice := common.FixedToPrice(fixedPrice)
+	require.Nil(err)
```

Comment:
This require statement should be called before calling `common.FixedToPrice()`. Same comment for the rest of the test cases in this file that place the require after `common.FixedToPrice()`.

Reply:
fixed in ec041c2d